trace_point.rb

```diff
@@ -396,7 +397,7 @@ def return_value
     Primitive.tracepoint_attr_return_value
   end
 
-  # Value from exception raised on the +:raise+ event
+  # Value from exception raised on the +:raise+ event, or rescued on the +rescue+ event.
```

Comment:
For consistency, either remove the colon before `raise`, or add it before `rescue`.